src/main/java/models/Player.java

```diff
@@ -1,7 +1,33 @@
 package models;
 
+import java.util.Scanner;
+
 /**
  * Created by nathan on 3/8/16.
  */
 public class Player extends User {
+    public int setBet;
+
```

Comment:
This should be changed to bet not setBet. You typically don't have variables with the same as method calls. Please change.